crypto/fipsmodule/bn/montgomery.c

```diff
@@ -156,3 +156,17 @@ int bn_from_montgomery_in_place(BN_ULONG r[], size_t num_r, BN_ULONG a[],
   }
   return 1;
 }
+
+#if !defined(OPENSSL_X86) && !defined(OPENSSL_X86_64) && \
+    !defined(OPENSSL_ARM) && !defined(OPENSSL_AARCH64)
+void bn_mul_mont(BN_ULONG *rp, const BN_ULONG *ap, const BN_ULONG *bp,
```

Comment:
This would need to have the prefixing stuff done to it in build.rs.

Reply:
As far as I can see, bn_mul_mont is already in SYMBOLS_TO_PREFIX in build.rs ?

Reply:
Thanks for pointing that out.